stdlib/@python2/abc.pyi

```diff
@@ -18,7 +18,7 @@ class ABCMeta(type):
     def __init__(self, name: str, bases: Tuple[type, ...], namespace: Dict[Any, Any]) -> None: ...
     def __instancecheck__(cls: ABCMeta, instance: Any) -> Any: ...
     def __subclasscheck__(cls: ABCMeta, subclass: Any) -> Any: ...
-    def _dump_registry(cls: ABCMeta, *args: Any, **kwargs: Any) -> None: ...
+    def _dump_registry(cls: ABCMeta, file: Optional[SupportsWrite[Any]] = ...) -> None: ...
```

Comment:
Nit: We now prefer the newer union syntax.
```suggestion
    def _dump_registry(cls: ABCMeta, file: SupportsWrite[Any] | None = ...) -> None: ...
```

Reply:
I prefer it too, but I copied this from the `print` definition. Do you think it's worth autofixing all the stubs?